assets/settings.json

```diff
@@ -16,6 +16,7 @@
 			"fullscreen": true,
 			"borderless": false,
 			"size": {"x": 1280, "y": 720},
+			//"pos": {"x": -1, "y": -1}, 		//Window appears centered if no position specified
```

Comment:
Does this actually parse successfully? I thought Cinder's JSON parser was pretty strict. Either way, I'd be a bit torn about leaving a comment in the JSON itself, since that's technically not supported in pure JSON. What do you think @adielfernandez ?

Reply:
Yup, it compiles and runs just fine. But I agree, it's not great practice, but I wanted to add it to start this conversation of how to let users know while keeping the default at a centered window.

Reply:
Yeah. I wonder if we should just set it to 0/0 by default (meaning the window would be top/left) and add a note in the readme to remove it if you want it to auto-center.

Reply:
That works, but I'd recommend something like 50, 50 so that the toolbar up top is on screen and people can drag the window where they need.

Reply:
Ah I see... Maybe we should just leave it out then? I feel like in most cases it should just work. Even if we wanted to position a window at -1/-1, that should happen automatically if the window is centered and +2px wide/tall.

Reply:
Sounds good to me 👍